packages/two_dimensional_scrollables/example/pubspec.yaml

```diff
@@ -6,8 +6,8 @@ publish_to: 'none'
 version: 1.0.0+1
 
 environment:
-  sdk: '>=3.1.0 <4.0.0'
-  flutter: ">=3.13.0"
+  sdk: '>=2.19.0 <4.0.0'
+  flutter: ">=3.7.0"
```

Comment:
This package won't work before Flutter 3.13
It uses new APIs that landed in 3.13